### PR TITLE
Feature/header v3

### PR DESCRIPTION
**Pull request by ccedrone**
re #305

### CODE REVIEW

**Review by silvalaura — APPROVED** [review]

**Review by orion-cengage — CHANGES_REQUESTED** [review]
1. The github nav item on the right should be using a link/button component so the icon uses the primary color and you get hover/active states.
image

File: website/react-magma-docs/src/components/Masthead/index.js [hunk 1]
```diff
@@ -1,18 +1,19 @@
 import React from 'react';
 import { Link } from 'gatsby';
-import { Logo } from '../Logo';
+import { Logo, SmallLogo } from '../Logo';
+import { ButtonVariant, IconButton } from 'react-magma-dom';
 import styled from '@emotion/styled';
 import { Header } from '@cengage-patterns/header';
-import { magma, useMediaQuery, AppBarPosition, Spacer } from 'react-magma-dom';
+import { magma, AppBarPosition } from 'react-magma-dom';
```
Comment (greenervigil) [inline website/react-magma-docs/src/components/Masthead/index.js:7]: line 4 & 7 can go together.

Comment (silvalaura) [reply website/react-magma-docs/src/components/Masthead/index.js]: addressed in [ff537d3](https://github.com/cengage/react-magma/pull/645/commits/ff537d347b7a929a91636d72b3fddc05696d520e)

**Review by orion-cengage — CHANGES_REQUESTED** [review]
1. Because the header is now using a white background, the focus state on the logo/home button needs to be updated to the focus state for light backgrounds.
image

2. Same thing with the "Skip Navigation" button.  The style needs to be updated and needs to be positioned with the boundaries of the header.
image

**Review by orion-cengage — CHANGES_REQUESTED** [review]
Looks like in the last update, the color of the text in the header's logo got changed to a lighter gray.  This should be neutral-700.

Also, please remove the change in color that happens on hover.

image

**Comment by ccedrone** [comment]
I resoundingly approve of this impressive PR!

**Pull request merged**